handler.go

```diff
@@ -60,7 +60,7 @@ func (h packageHandler) Handle(w http.ResponseWriter, r *http.Request, ps httpro
 	}{
 		Repo:         h.pkg.Repo,
 		CanonicalURL: canonicalURL,
-		GodocURL:     fmt.Sprintf("https://godoc.org/%s%s", canonicalURL, ps.ByName("path")),
+		GodocURL:     fmt.Sprintf("https://%s/%s%s", h.config.Godoc.Host, canonicalURL, ps.ByName("path")),
```

Comment:
Should we just default to whatever the protocol (eg http/https) that users specify in `.Host`?
ie Is it safe to assume that this is always `https`?

Reply:
nvm, didn't read above comments

Reply:
We don't support adding the scheme in the config and sanitize it. This should actually probably be the scheme used in the incoming request instead.